src/java/org/apache/cassandra/db/lifecycle/DefaultLogAwareFileLister.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We have not actually donated this code to the ASF yet, so I don't think this header is right?

Reply:
should I Datastax License?